rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -258,8 +262,40 @@ def _decode_curve_events(
 
         return None, []
 
-    @staticmethod
+    def _decode_curve_gauge_events(
+            self,
+            tx_log: EvmTxReceiptLog,
+            transaction: EvmTransaction,  # pylint: disable=unused-argument
+            decoded_events: list['EvmEvent'],
+            all_logs: list[EvmTxReceiptLog],  # pylint: disable=unused-argument
+            action_items: Optional[list[ActionItem]],  # pylint: disable=unused-argument
+    ) -> tuple[Optional['EvmEvent'], list[ActionItem]]:
+        if tx_log.topics[0] in (GAUGE_DEPOSIT, GAUGE_WITHDRAW):
+            provider = hex_or_bytes_to_address(tx_log.topics[1])
+            gauge_address = tx_log.address
+            raw_amount = hex_or_bytes_to_int(tx_log.data)
+            for event in decoded_events:
+                crypto_asset = event.asset.resolve_to_crypto_asset()
+                if (
+                    event.location_label == provider and
+                    event.address == gauge_address and
+                    event.balance.amount == asset_normalized_value(amount=raw_amount, asset=crypto_asset)  # noqa: E501
+                ):
+                    if tx_log.topics[0] == GAUGE_DEPOSIT:
+                        event.event_type = HistoryEventType.DEPOSIT
+                        event.event_subtype = HistoryEventSubType.DEPOSIT_ASSET
+                        event.notes = f'Deposit {event.balance.amount} {crypto_asset.symbol} into curve gauge'  # noqa: E501
+                        event.counterparty = CPT_CURVE
+                    else:  # Withdraw
+                        event.event_type = HistoryEventType.WITHDRAWAL
+                        event.event_subtype = HistoryEventSubType.REMOVE_ASSET
+                        event.notes = f'Deposit {event.balance.amount} {crypto_asset.symbol} from curve gauge'  # noqa: E501
```

Comment:
copy pasta. Also same comment as above

```suggestion
                        event.notes = f'Withdraw {event.balance.amount} {crypto_asset.symbol} from curve gauge'  # noqa: E501
```